docs/the-fleetshard-pattern.md

```diff
@@ -0,0 +1,45 @@
+This document contains useful links on the fleetshard pattern.
+
+# Why the fleetshard model? 
```

Comment:
is it expected that it is in markdown? Vs asciidoc?

Reply:
Markdown was the first go-to format that I went with when creating the file. 
Are the files expected to be in Asciidoc?

Reply:
Good points @grdryn 

> Markdown was the first go-to format that I went with when creating the file.

The majority of the [docs](https://github.com/bf2fc6cc711aee1a0c2a/ffm-fleet-manager-go-template/tree/main/docs) in this repo are in markdown that's why I went with the format. I am leaning toward keeping it that way and we could create an issue to change all the docs to asciidoc format.